zone/attack.cpp

```diff
@@ -4195,11 +4223,23 @@ void Mob::CommonDamage(Mob* attacker, int64 &damage, const uint16 spell_id, cons
 				spells[spell_id].name /* Message4 */
 			);
 		}
-	} //end packet sending
+		// If a bot is the attacker, send a damage message ot the Bot Owner
+		else if (attacker->GetTarget() && spell_id != SPELL_UNKNOWN && attacker->IsBot() && damage > 0 && !Critical && attacker && attacker != this && RuleB(Bots, DisplaySpellDamage)) {
```

Comment:
Have the IsBot check first that way a majority of the time this check ends earlier

Reply:
updated